src/Workspaces/Core/Portable/Workspace/Solution/ProjectDependencyGraph_RemoveProject.cs

```diff
@@ -73,12 +77,15 @@ private static ImmutableDictionary<ProjectId, ImmutableHashSet<ProjectId>> Compu
             // 'projectId' from the set of references if it exists.
             foreach (var (id, _) in existingForwardReferencesMap)
             {
-                builder.MultiRemove(id, removedProjectId);
+                foreach (var removedProjectId in removedProjectIds)
```

Comment:
In this case a MultiRemove that takes multiple would be great.

Reply:
I'd expect it to be more performant, at least when removedProjectIds has mutliple things in it, since we'll only do one creation of a new ImmutableHashSet and one update in the ImmutableDictionary.Builder rather than n.